### DISCUSSION TITLE
Suggestion: Include mention of Drawer in Sheet component for more intuitive naming

### DISCUSSION THREAD

**Discussion by nicolaerusan** [shadcn-ui/ui#2082] [General]
I was searching the library for what I typically think of as a Drawer component and thought it didn't exist. Eventually found my way to the Sheet component, but I think that's a non-intuitive name for what most people expect that to be called.

When I see Sheet, my first thought is some sort of Grid or Spreadsheet like component.

Just some feedback from someone that was a bit surprised by the naming :)

**Comment by onurhan1337** [comment]
I think the **_drawer_** component was another plan. Here is new: https://ui.shadcn.com/docs/components/drawer
